e2e/tests-dfx/build.bash

```diff
@@ -9,11 +9,40 @@ setup() {
 }
 
 teardown() {
+    stop_webserver
     dfx_stop
 
     standard_teardown
 }
 
+@test "can build a custom canister with wasm from a url" {
+    install_asset wasm/identity
+    mkdir -p www/wasm
+    mv main.wasm www/wasm/
+    mv main.did www/wasm
+    start_webserver --directory www
+    dfx_start
+
+    dfx_new
+
+    jq '.canisters={}' dfx.json | sponge dfx.json
+
+    # URL for candid field is not supported
+    # shellcheck disable=SC2154
+    cp "${assets}/wasm/identity/main.did" main.did
+    # jq '.canisters.e2e_project.candid="http://localhost:'"$E2E_WEB_SERVER_PORT"'/wasm/main.did"' dfx.json | sponge dfx.json
+    jq '.canisters.e2e_project.candid="main.did"' dfx.json | sponge dfx.json
+
+    jq '.canisters.e2e_project.wasm="http://localhost:'"$E2E_WEB_SERVER_PORT"'/wasm/main.wasm"' dfx.json | sponge dfx.json
```

Comment:
Only that remote canister support makes it more complicated.  So I thought I would start with this in order to get some traction.  I wouldn't mind leaving this as a draft until I can accompany it with support for URLs for candid files.

Reply:
Sounds good. Instead of making this a draft and hold this feature we could also simply add an explanation to the changelog that support for did files is coming soon, too.